tasks/build-ext.js

```diff
@@ -15,11 +16,22 @@ var buildDir = path.join(root, 'build', 'ol.ext');
  * @return {Array.<Object>} Array of objects representing external modules.
  */
 function getExternalModules() {
-  return pkg.ext.map(function(name) {
-    return {
-      name: name,
-      main: require.resolve(name)
-    };
+  return pkg.ext.map(function(item) {
+    if (typeof item === 'string' || item instanceof String) {
```

Comment:
There is a difference between creating a string by calling the constructor and by initiating with a value.

	var s = 'abc'
	typeof s
	> "string"

	var t = new String('abc')
	typeof t
	> "object"
	t instanceof String
	> true